gdx/src/com/badlogic/gdx/utils/Array.java

```diff
@@ -298,8 +298,8 @@ public void removeRange (int start, int end) {
 		if (ordered)
 			System.arraycopy(items, start + count, items, start, size - (start + count));
 		else {
-			for (int i = 0; i < count; i++)
-				items[start + i] = items[lastIndex + i];
+			if (count >= 0)
+				System.arraycopy(items, lastIndex + 0, items, start + 0, count);

```

Comment:
Count will never be negative and the `+0` are weird, otherwise ok. (It surely is more straightforward copy than the branch above.)